### PR TITLE
Markup and style standards

### PR DESCRIPTION
**Pull request by clodpated**
Beginning the standards for markup and style. 

* Created a standards md file that gives lists of basic rules
* Created sample files as well, commenting them where appropriate

### CODE REVIEW

File: getting-started/standards/markup.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <!-- Include a valid viewport tab for device compatibility on responsive pages -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Include this line for IE compatibility -->
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+
+    <title>Document</title>
+    
+    <!-- Include your CSS in the head, avoiding multiple files where possible. Do not use `<style>` elements for inline CSS -->
+    <link rel="stylesheet" href="style.css">
```
Comment (jcontonio) [inline getting-started/standards/markup.html]: In standards bullets you say to close with `/>` for bits liek this. Let's keep that standard in the html files for meta, script, etc if that's the suggestion.

File: getting-started/standards/markup.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    <!-- Only link to javascript in the head when required, like with Modernizr or polyfills -->
+
+  </head>
+
+  <body>
+    
+    <!-- When creating forms, use this pattern for accessibility -->
+    <label for="some-input">Input:</label>
+    <input type="text" id="some-input" placeholder="Your text" />
```
Comment (jcontonio) [inline getting-started/standards/markup.html]: Let's wrap this in a form tag so it's compliant.

File: getting-started/standards.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+* Use semantic elements and aria roles wherever possible. Remember that most roles are already defined if you use the correct element https://www.w3.org/TR/html-aria/#sec-strong-native-semantics
+* Avoid using character entities, instead using proper UTF characters. Exceptions include `&nbsp;`, `&amp;`, and `&gt;&lt;`. 
+* Make liberal use of tools like Jekyll, Fathom Editor, React, etc to break markup into manageable, reusable bits. 
+
+## CSS / SCSS
+* SCSS is the standard at Mindspace
+* All styles, variables, and mixins should be included to a root file, `style.scss`. 
+* Include [Bourbon](http://bourbon.io/) and [Neat](http://neat.bourbon.io/) for vendor prefixing, media queries, and grid
+* Include [Normalize](https://necolas.github.io/normalize.css/) rather than a CSS reset
+* Break variables, common elements, and template-specific scss into smaller modules. A typical css folder may look something like:
```
Comment (jcontonio) [inline getting-started/standards.md]: Let's add alphabetize your rules, as well as the order in which we do things...which is usually:

```
.class {
  default class rules
  media queries
  &.things
    media queries
  nested shit
```

File: getting-started/standards.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# Code standards and best practices
+
+## General
+* Indent with 2 spaces
+* Nested lines should be indented once
+* Comments and whitespace are your friends
```
Comment (jcontonio) [inline getting-started/standards.md]: Let's link to [DocBlockr](https://github.com/spadgos/sublime-jsdocs) here.

Comment (jcontonio) [inline getting-started/standards.md]: And maybe some tips on using it

File: getting-started/standards/markup.html (hunk 1 above)
Comment (clodpated) [inline getting-started/standards/markup.html]: Haha whoops

**Pull request merged**